app/templates/teachers/teacher-student-view.jade

```diff
@@ -0,0 +1,220 @@
+extends /templates/base-flat
+
+block page_nav
+  include ../courses/teacher-dashboard-nav.jade
+
+block content
+
+  if !view.user
+    div Loading
+  else
+    .container
+      +breadcrumbs
+      if !me.isAnonymous() && me.isTeacher()
+      - var isOwner = view.classroom ? view.classroom.get('ownerID') === me.id : false;
+        // - var validStudent = ...  
+        // load all students in this class
+        // check if this student is in the class
+          
+      if isOwner
+        // also check if validStudent 
+        h3(class="m-t-2")
+          span(data-i18n="teacher.student_profile")
+          span.spr :
+          span= view.user.get('name')
+        // this user object doesn't have an email property
+        h4(class="student-details" data-i18n="teacher.student_overview")
+        if (view.user.get('email'))
+          div(class="small-details")
+            span(data-i18n="teacher.student_email")
+            span.spr :
+            span= view.user.get('email')
+        else
+          div(class="small-details") 
+            i(data-i18n="teacher.no_email")
+        div(class="small-details")
+          span(data-i18n="user.last_played")
+          span.spr :
+          span= view.lastPlayedString
+          
+        div(class="small-details") 
+          - var status = view.user.prepaidStatus()
+          span(data-i18n='view.user.status')
+          span.spr
+          span(data-i18n="teacher.license_status")
+          span.spr : 
+          strong(class= status === 'expired' ? 'text-danger' : '')= view.studentStatusString()
+        //- 
+        //- svg#visualisation(width="1000", height="520")
+        //- 
+        
+        div.playtime-charts
+          // i18n needed
+          h4(data-i18n="teacher.playtime_detail")
+        
+          .graphsSelector
+            span(data-i18n="teacher.select_course")
+            span :
+            span.spr
+            select#course-dropdown.text-navy
+              each versionedCourse in view.classroom.get('courses') || []
+                - var course = _.find(view.courses.models, function(c) {return c.id === versionedCourse._id;});
+                if !course
+                  - continue;
+                - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
+                - if ((instance && instance.hasMember(view.user)))
+                  option(value=course.id)=course.get('name')
+            div.legend
+              svg(width=15, height=15, style="margin: 0 5px 0 0")
+                rect(width=15, height=15,fill="rgb(32, 87, 43)")
+              span(data-i18n="teacher.student_completed") 
+              
+              svg(width=15, height=15, style="margin: 0 5px 0 15px")
+                rect(width=15, height=15,fill="rgb(242, 190, 25)")
+              span(data-i18n="teacher.student_in_progress") 
+              
+              svg(width=15, height=15 style="margin: 0 5px 0 15px")
```

Comment:
Prefer not to have inline styles for the most part, but in this case it might be justifiable.

I try to keep whether I use commas in attribute lists locally consistent (and keep spaces after them). Not super important.